server/src/main/java/org/opensearch/index/MergePolicyConfig.java

```diff
@@ -41,6 +42,9 @@
 import org.opensearch.core.common.unit.ByteSizeUnit;
 import org.opensearch.core.common.unit.ByteSizeValue;
 
+import static org.apache.lucene.index.LogMergePolicy.DEFAULT_MAX_MERGE_DOCS;
+import static org.apache.lucene.index.LogMergePolicy.DEFAULT_NO_CFS_RATIO;
+
 /**
  * A shard in opensearch is a Lucene index, and a Lucene index is broken
  * down into segments. Segments are internal storage elements in the index
```

Comment:
This class documentation is quite comprehensive for the current behavior. Can you update it accordingly for your changes? I think it will help me review this change. Specifically, I'd like to understand the behavior of the current merge-related settings with regard to the changes here. For example, do the existing settings apply for data streams if `index.datastream_merge.policy` is set to `tiered`, but the new settings apply if set to `log_byte_size`?

Reply:
Thanks for reviewing. I will update the documentation, thanks for pointing it out. 
Your understanding is correct, this change isn't intended to change anything with current default (i.e. tiered) merge policy. User will have to explicitly set it to `log_byte_size` using `index.datastream_merge.policy` setting.